### DISCUSSION TITLE
Next.js image import vs Storybook image imports

### DISCUSSION THREAD

**Discussion by gregorybolkenstijn** [storybookjs/storybook#15679] [Help]
With Next.js 11 we can now import images like this:

`import someImage from './some-folder/some-image.png'`

`someImage` will be an object of this type:

```
interface StaticImageData {
    src: string;
    height: number;
    width: number;
    blurDataURL?: string;
}
```
See: https://nextjs.org/blog/next-11#automatic-size-detection-local-images

Storybook's file loader will import the same file but `someImage` will just be a string.

I can handle this difference in my own code, but I'd rather have image imports work the same in every context (Next.js and Storybook). How can I change Storybooks' image loader in the Webpack config so it will return an object with a src property as well? Other suggestions for a solution are also welcome!

**Comment by shilman** [comment]
Please track this issue on NextJS and comment if there are pieces missing https://github.com/vercel/next.js/issues/18393
